fail2ban/tests/actiontestcase.py

```diff
@@ -71,6 +71,12 @@ def testSubstituteRecursiveTags(self):
 									'ABC': '123 192.0.2.0',
 									'xyz': '890 123 192.0.2.0',
 								})
+		# obscure embedded case
+		self.assertEqual(CommandAction.substituteRecursiveTags({'A': '<<PREF>HOST>', 'PREF': 'IPV4'}),
+						 {'A': '<IPV4HOST>', 'PREF': 'IPV4'})
```

Comment:
>  > +             # obscure embedded case
>  > +             self.assertEqual(CommandAction.substituteRecursiveTags({'A': '<<PREF>HOST>', 'PREF': 'IPV4'}),
>  > +                                              {'A': '<IPV4HOST>', 'PREF': 'IPV4'})

>    Agree with this one, but not with one bellow ...

below was somewhat an exercise toward one of the viable approaches on
who to implement support for IPv6 while avoiding duplication ;)

but yeah -- since it wasn't supported before -- fine with me if this one
remains "supported" and then eventually we might support embedded ones